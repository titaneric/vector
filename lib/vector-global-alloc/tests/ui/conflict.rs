//! Shared UI test to ensure only one global allocator can be defined.

extern crate vector_global_alloc;

// Should conflict with the declaration from the `vector_global_alloc`.
#[global_allocator]
static ALLOC: std::alloc::System = std::alloc::System;

fn main() {}
