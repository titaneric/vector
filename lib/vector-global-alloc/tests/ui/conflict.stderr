error: the `#[global_allocator]` in this crate conflicts with global allocator in: vector_global_alloc
